### PR TITLE
OWC18 Teams update

### PR DESCRIPTION
**Pull request by juankristal**

### CODE REVIEW

File: wiki/Tournaments/OWC/2018/en.md [hunk 1]
```diff
@@ -54,42 +54,47 @@ The osu! World Cup 2018 is run by various community members by distributing the
...
+| ![][flag_RO] | **Romania** | **[badeu](https://osu.ppy.sh/users/1473890)**, [vuru](https://osu.ppy.sh/users/7432712), [\_thicke](https://osu.ppy.sh/users/8334445), [Chiu](https://osu.ppy.sh/users/3148900), [KromaRO](https://osu.ppy.sh/users/7543723), [eternum](https://osu.ppy.sh/users/4581069), [roliy](https://osu.ppy.sh/users/9578404), [Rohulk](https://osu.ppy.sh/users/3219026) |
+| ![][flag_RU] | **Russian Federation** | **[follon](https://osu.ppy.sh/users/3973474)**, [AxewB](https://osu.ppy.sh/users/4928776) [Alumetri](https://osu.ppy.sh/users/5371497), [Red\_Pixel](https://osu.ppy.sh/users/4170932), [Aden](https://osu.ppy.sh/users/4342841), [neko ds](https://osu.ppy.sh/users/4688735), [Okinotori](https://osu.ppy.sh/users/4346274), [talala](https://osu.ppy.sh/users/1389663) |
+| ![][flag_SK] | **Slovakia** | **[Hranolka](https://osu.ppy.sh/users/6149947)**, [AtHeoN](https://osu.ppy.sh/users/1770367), [andrejus77](https://osu.ppy.sh/users/2463937), [SneakY NickY](https://osu.ppy.sh/users/8847379), [Tikef](https://osu.ppy.sh/users/9149213), [PemiX](https://osu.ppy.sh/users/6974470), [Im sorry](https://osu.ppy.sh/users/1285945), [Nikolas](https://osu.ppy.sh/users/7759641) |
```
Comment (WalterToro) [inline wiki/Tournaments/OWC/2018/en.md]: Im sorry > PeteX

File: wiki/Tournaments/OWC/2018/en.md [hunk 2]
```diff
@@ -54,42 +54,47 @@ The osu! World Cup 2018 is run by various community members by distributing the
...
+| ![][flag_TW] | **Taiwan** | **[Shiina Noriko](https://osu.ppy.sh/users/1285637)**, [Flask](https://osu.ppy.sh/users/959763), [\_Shield](https://osu.ppy.sh/users/1860489), [\[ Zane \]](https://osu.ppy.sh/users/3517706), [CrtReXn](https://osu.ppy.sh/users/1810239), [Rizer](https://osu.ppy.sh/users/5155973), [DazzLE_Wind](https://osu.ppy.sh/users/2537924), [GfMRT](https://osu.ppy.sh/users/3163649) |
+| ![][flag_TH] | **Thailand** | **[- Phantasma -](https://osu.ppy.sh/users/1427407)**, [Applause](https://osu.ppy.sh/users/1832711), [NonxE](https://osu.ppy.sh/users/319312), [Deppyforce](https://osu.ppy.sh/users/5286213), [Mikkuri](https://osu.ppy.sh/users/317494), [KiraMoe](https://osu.ppy.sh/users/3504692), [redcrosssix](https://osu.ppy.sh/users/2554965), [VanillaYuki](https://osu.ppy.sh/users/5332553) |
```
Comment (WalterToro) [inline wiki/Tournaments/OWC/2018/en.md]: KiraMoe > Chorus

File: wiki/Tournaments/OWC/2018/en.md [hunk 3]
```diff
@@ -54,42 +54,47 @@ The osu! World Cup 2018 is run by various community members by distributing the
...
+| ![][flag_GB] | **United Kingdom** | **[Bubbleman](https://osu.ppy.sh/users/5182050)**,  [Doomsday](https://osu.ppy.sh/users/18983), [OPJames](https://osu.ppy.sh/users/4117142), [Spare](https://osu.ppy.sh/users/2204373), [MoeYandere](https://osu.ppy.sh/users/2565902), [SoraRoxas01](https://osu.ppy.sh/users/1986262), [Karthy](https://osu.ppy.sh/users/4196808), [Jameslike](https://osu.ppy.sh/users/2415743)|
+| ![][flag_US] | **United States** | **[Apraxia](https://osu.ppy.sh/users/4194445)**, [Toy](https://osu.ppy.sh/users/4919398), [Monko2k](https://osu.ppy.sh/users/4852013), [fieryrage](https://osu.ppy.sh/users/3533958), [DigitalHypno](https://osu.ppy.sh/users/4384207), [Vaxei](https://osu.ppy.sh/users/4787150), [he rack](https://osu.ppy.sh/users/2644828), [idke](https://osu.ppy.sh/users/4650315) |
```
Comment (WalterToro) [inline wiki/Tournaments/OWC/2018/en.md]: There is still the link for Ok xd instead of Toy

File: wiki/Tournaments/OWC/2018/en.md [hunk 4]
```diff
@@ -54,42 +54,47 @@ The osu! World Cup 2018 is run by various community members by distributing the
...
+| ![][flag_SK] | **Slovakia** | **[Hranolka](https://osu.ppy.sh/users/6149947)**, [AtHeoN](https://osu.ppy.sh/users/1770367), [andrejus77](https://osu.ppy.sh/users/2463937), [SneakY NickY](https://osu.ppy.sh/users/8847379), [Tikef](https://osu.ppy.sh/users/9149213), [PemiX](https://osu.ppy.sh/users/6974470), [Im sorry](https://osu.ppy.sh/users/1285945), [Nikolas](https://osu.ppy.sh/users/7759641) |
+| ![][flag_SG] | **Singapore** | **[M4-K1](https://osu.ppy.sh/users/5210595)**, [GSBlank](https://osu.ppy.sh/users/2312106), [Cleveland](https://osu.ppy.sh/users/5104320), [Rtyzen](https://osu.ppy.sh/users/2439822),  [Emilia](https://osu.ppy.sh/users/2003326), [Loslite](https://osu.ppy.sh/users/6398160), [\[-Lockon-\]](https://osu.ppy.sh/users/6726331), [Raindrop](https://osu.ppy.sh/users/1155871) |
```
Comment (Joehuu) [inline wiki/Tournaments/OWC/2018/en.md]: `Singapore` before `Slovakia`.

File: wiki/Tournaments/OWC/2018/en.md [hunk 5]
```diff
@@ -54,42 +54,47 @@ The osu! World Cup 2018 is run by various community members by distributing the
 
 ## Participants
 
+**Confirmed Rosters**
+
+| | Country | Members |
+| ---: | :---: | :--- |
```
Comment (Joehuu) [inline wiki/Tournaments/OWC/2018/en.md]: `| ---: | :---: | :--- |` -> `| --: | :-: | :-- |`

File: wiki/Tournaments/OWC/2018/en.md [hunk 6]
```diff
@@ -54,42 +54,47 @@ The osu! World Cup 2018 is run by various community members by distributing the
...
+| ![][flag_GB] | **United Kingdom** | **[Bubbleman](https://osu.ppy.sh/users/5182050)**,  [Doomsday](https://osu.ppy.sh/users/18983), [OPJames](https://osu.ppy.sh/users/4117142), [Spare](https://osu.ppy.sh/users/2204373), [MoeYandere](https://osu.ppy.sh/users/2565902), [SoraRoxas01](https://osu.ppy.sh/users/1986262), [Karthy](https://osu.ppy.sh/users/4196808), [Jameslike](https://osu.ppy.sh/users/2415743)|
+| ![][flag_US] | **United States** | **[Apraxia](https://osu.ppy.sh/users/4194445)**, [Toy](https://osu.ppy.sh/users/4919398), [Monko2k](https://osu.ppy.sh/users/4852013), [fieryrage](https://osu.ppy.sh/users/3533958), [DigitalHypno](https://osu.ppy.sh/users/4384207), [Vaxei](https://osu.ppy.sh/users/4787150), [he rack](https://osu.ppy.sh/users/2644828), [idke](https://osu.ppy.sh/users/4650315) |
+
 _Pending Rosters_
 
 | | Country | Members |
 | ---: | :---: | :--- |
```
Comment (Joehuu) [inline wiki/Tournaments/OWC/2018/en.md]: `| ---: | :---: | :--- |` -> `| --: | :-: | :-- |`

File: wiki/Tournaments/OWC/2018/en.md [hunk 7]
```diff
@@ -54,42 +54,42 @@ The osu! World Cup 2018 is run by various community members by distributing the
 
 ## Participants
 
-_Pending Rosters_
+**Confirmed Rosters**
```
Comment (WalterToro) [inline wiki/Tournaments/OWC/2018/en.md]: Delete this line and we are ready

**Pull request merged**